straxen/url_config.py

```diff
@@ -207,11 +208,15 @@ def fetch(self, plugin):
         # will become the method kwargs
         url, url_kwargs = self.split_url_kwargs(url)
 
+
         kwargs = {k: self.fetch_attribute(plugin, v)
                   for k, v in url_kwargs.items()}
 
         # construct a deterministic hash key
-        key = strax.deterministic_hash((url, kwargs))
+        try:
+            key = strax.deterministic_hash((url, kwargs))
+        except TypeError:
+            key = uuid4()
```

Comment:
I don't like the look of this, can you explain where/why it's needed?

Reply:
This is to allow non-hashable kwargs such as functions.

Reply:
This is to allow non-hashable kwargs such as functions.

Reply:
Sure, i assumed whomever put it there wanted it to be in a single place so changes are synched between the two plugins that use this default value

Reply:
simplified to be easier to follow logic